docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    command: >
+      sh -c "pyton manage.py makemigrations &&
```

Comment:
are you sure it works? :)
what is pyton?
why do you need to run makemigrations if you always push them in github?

Reply:
Also all tests should be passed